include/deal.II/distributed/tria.h

```diff
@@ -953,6 +997,13 @@ namespace parallel
       void attach_mesh_data();
 
       /**
+       * Internal function notifying all registered classes to provide their
+       * weights before repartitioning occurs. Called from
+       * execute_coarsening_and_refinement().
+       */
+      void get_cell_weights(std::vector<unsigned int> &weights);
```

Comment:
Can you document the intent of the argument (e.g., its size, what it represents, etc)?